src/DIPS.Xamarin.UI/Controls/Toast/ToastOptions.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Threading.Tasks;
+using DIPS.Xamarin.UI.Internal.Xaml;
+using Xamarin.Forms;
+
+namespace DIPS.Xamarin.UI.Controls.Toast
+{
+    /// <summary>
+    ///     Options to setup in the Toast control
+    /// </summary>
+    public class ToastOptions : BindableObject
```

Comment:
Dont think it needs to inherit from BindableObject

Reply:
Oh, same residue... missed to remove the extend after removing the bindings